tests/extendrtests/tests/testthat/test-memory_leaks.R

```diff
@@ -1,58 +1,50 @@
-
-
-
 test_that("extendr conversions/panics does not leak", {
-
   ## get list of rust_functions and value generator functions to test.
-  e_pkg = environment(leak_implicit_doubles)
-  fnames = c(
-
-    "leak_implicit_doubles",# single burn-in function
-
-    "leak_implicit_doubles",
-    "leak_implicit_strings",
-
+  e_pkg <- environment(leak_arg2_try_implicit_doubles)
```

Comment:
This test had something specific with the conversion types we had. Now it only test one thing. The formatter went crazy on it. But it passes, with minor changes, so 🤷